src/Sulu/Bundle/ResourceBundle/Resource/FilterManager.php

```diff
@@ -467,8 +507,28 @@ public function getFeaturesForContext($context)
     }
 
     /**
+     * Removes conditions from condition groups when they are not in the given array
+     *
+     * @param ConditionGroupEntity $conditionGroup
+     * @param array $conditionIds
+     */
+    protected function removeNonExistentConditions(
+        $conditionGroup,
+        $conditionIds
+    ) {
+        foreach ($conditionGroup->getConditions() as $condition) {
+            if ($condition->getId() && array_search($condition->getId(), $conditionIds) === false) {
```

Comment:
It would probably be better to use `in_array` instead of `array_search` if you don't use the resulting key.